maven-metadata-inferer/src/main/java/org/ndx/aadarchi/inferer/maven/enhancers/ContainerEnhancer.java

```diff
@@ -32,9 +35,35 @@ protected Component getContainedElementWithName(String key) {
 	}
 
 	@Override
-	protected void containedDependsUpon(Component contained, Component found, String string) {
-		contained.uses(found, string);
+	protected String getRelationshipDescription(Component contained, Component found) {
```

Comment:
Can you add the javadoc, because even if I roughly understand the process, some details are quite puzzling to me.